.github/dependabot.yml

```diff
@@ -0,0 +1,31 @@
+version: 2
+updates:
+
+  - package-ecosystem: "gomod"
+    directory: "/"
+    schedule:
+      interval: "daily"
+      time: "08:00"
+      timezone: "America/Los_Angeles"
+    target-branch: "main"
+    commit-message:
+      prefix: "[golang-version]"
+      include: "scope"
+    pull-request-branch-name:
+      separator: "-"
+    open-pull-requests-limit: 8
+    reviewers:
+      - "Layr-Labs/eigenda-intg"
+    labels:
+      - "version"
+      - "golang"
+    allow:
+      - dependency-type: "direct"
+    groups:
+      golang-version-updates:
```

Comment:
```suggestion
      # Creates one consolidated PR for all minor/patch updates to reduce PR noise
      # Major version updates (e.g., 1.x.x -> 2.x.x) are excluded since they might contain breaking changes and should be reviewed separately.
      golang-version-updates:
```